app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
I was thinking about that too, but finally dismissed the idea. The dates are enclosed by spans which in turn are specially formatted. I could either drop the special formatting (which I think is a bad idea) or I could include HTML into a __()-Expression which is even worse ;-)
I added a comma, however due to the aforementioned reasons I refrained from using an atomic expression.